capstone/src/test/java/com/google/sps/servlets/PollServletTest.java

```diff
@@ -109,6 +141,8 @@ public void tearDown() {
 
   @Test
   public void doGet_userLoggedIn() throws IOException {
+    this.addOptionsToGroup();
+    when(Group.getGroupEntity(mockRequest, mockResponse, datastore)).thenReturn(GROUP_ENTITY);
     pollServlet.doGet(mockRequest, mockResponse);
```

Comment:
Can you put new lines before and after the `.doGet` call so you get a three block

```java
// Arrange

// Act

// Assert
```

structure to your tests?